### PR TITLE
i18n : integrate FR JA ZH translation

### PR DESCRIPTION
**Pull request by akhabali**
**What is the current behavior?** (You can also link to an open issue here)

**What is the new behavior?**

**Please check if the PR fulfills these requirements**

- [ ] The commit message follows Talend standard
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features) ?
- [ ] The code coverage on new code >75%
- [ ] The new code does not introduce new technical issues (sonar / eslint)

**What kind of change does this PR introduce?**

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build / CI related changes
- [ ] Other... Please describe:

**Does this PR introduce a breaking change?**

- [ ] Yes
- [ ] No

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**:

### CODE REVIEW

**Comment by build-talend-common** [comment]
Pull request test result: [success](https://ci-common.datapwn.com/job/components_pull-request_master/17/) [:star:](https://ci-common.datapwn.com/job/components_pull-request_master/17/) 
[Sonar results](https://sonar.datapwn.com/overview?id=org.talend.components%3Acomponents%3Aakhabali%2Fi18n%2F7mai19)

**Comment by build-talend-common** [comment]
Pull request test result: [failure](https://ci-common.datapwn.com/job/components_pull-request_master/21/) [:x:](https://ci-common.datapwn.com/job/components_pull-request_master/21/)

File: core/components-common-oauth/src/main/resources/org/talend/components/common/oauth/properties/messages_fr.properties [hunk 1]
```diff
@@ -12,7 +12,7 @@ property.audience.displayName=Public vis\u00E9
 
 # Implicit Flow
 property.clientId.displayName=ID du client
-property.clientSecret.displayName=Secret du client
+property.clientSecret.displayName=Client Secret
```
Comment (sbouyer) [inline core/components-common-oauth/src/main/resources/org/talend/components/common/oauth/properties/messages_fr.properties]:
```suggestion
property.clientSecret.displayName=Secret du client
```

File: components/components-azurestorage/src/main/resources/org/talend/components/azurestorage/queue/tazurestoragequeueinputloop/messages_fr.properties [hunk 2]
```diff
@@ -1,2 +1,2 @@
 component.tAzureStorageQueueInputLoop.title=Lire des messages depuis une file dans Microsoft Azure Storage sans limite
-property.loopWaitTime.displayName=Temps d''attente de la boucle (secondes)
+property.loopWaitTime.displayName=Temps d'attente de la boucle (secondes)
```
Comment (gonchar-ivan) [inline components/components-azurestorage/src/main/resources/org/talend/components/azurestorage/queue/tazurestoragequeueinputloop/messages_fr.properties]: please, fix
d'attente ->  d''attente

File: components/components-azurestorage/src/main/resources/org/talend/components/azurestorage/table/tazurestorageinputtable/messages_fr.properties [hunk 3]
```diff
@@ -1,4 +1,4 @@
-component.tAzureStorageInputTable.title=Lire les lignes d''une table dans Microsoft Azure Storage
+component.tAzureStorageInputTable.title=Lire les lignes d'une table dans Microsoft Azure Storage
```
Comment (gonchar-ivan) [inline components/components-azurestorage/src/main/resources/org/talend/components/azurestorage/table/tazurestorageinputtable/messages_fr.properties]: please, add 1 one more quote here

File: components/components-fileio/simplefileio-definition/src/main/resources/org/talend/components/simplefileio/s3/S3DatasetProperties_fr.properties [hunk 4]
```diff
@@ -2,19 +2,19 @@ form.Main.title=Propri\u00E9t\u00E9s du jeu de donn\u00E9es S3
 form.Main.displayName=Propri\u00E9t\u00E9s du jeu de donn\u00E9es S3
 
 property.bucket.displayName=Bucket
-property.object.displayName=Chemin d''acc\u00E8s
-property.encryptDataInMotion.displayName=Chiffrer les donn\u00E9es \u00E0 la vol\u00E9e
+property.object.displayName=Chemin d'acc\u00E8s
```
Comment (gonchar-ivan) [inline components/components-fileio/simplefileio-definition/src/main/resources/org/talend/components/simplefileio/s3/S3DatasetProperties_fr.properties]: I suggest to fix it:
```suggestion
property.object.displayName=Chemin d''acc\u00E8s
```

File: components/components-googledrive/components-googledrive-runtime/src/main/resources/org/talend/components/google/drive/runtime/messages_fr.properties [hunk 5]
```diff
@@ -33,19 +33,19 @@ error.validation.folder.empty=Le nom du dossier ne peut pas \u00EAtre vide.
 error.validation.put.invalid.flow=Flow invalide, v\u00E9rifiez les connecteurs et propri\u00E9t\u00E9s.
 error.validation.local.filename.empty=Le nom du fichier local ne peut \u00EAtre vide.
 error.validation.destinationFolderName.empty=Le nom du dossier de destination ne peut pas \u00EAtre vide.
-error.validation.connection.datastorePath.empty=Le chemin d''acc\u00E8s au DataStore ne peut \u00EAtre vide. D\u00E9finissez-le dans l''onglet Avanc\u00E9 de Connexion.
+error.validation.connection.datastorePath.empty=Le chemin d''acc\u00E8s au Datastore ne peut \u00EAtre vide. Configurez-le dans l'onglet Avanc\u00E9 de Connexion.
 error.validation.connection.proxyHost.empty=L''h\u00F4te du proxy ne peut \u00EAtre vide.
-error.validation.connection.proxyPort.invalid=Le port du proxy est invalide.
+error.validation.connection.proxyPort.invalid=Le port du proxy n''est pas valide.
 error.validation.connection.sslAlgorithm.empty=L'algorithme SSL ne peut \u00EAtre vide.
```
Comment (gonchar-ivan) [inline components/components-googledrive/components-googledrive-runtime/src/main/resources/org/talend/components/google/drive/runtime/messages_fr.properties]: add one more single quote
```suggestion
error.validation.connection.sslAlgorithm.empty=L''algorithme SSL ne peut \u00EAtre vide.
```

File: components/components-jdbc/components-jdbc-definition/src/main/resources/org/talend/components/jdbc/datastore/messages_fr.properties [hunk 6]
```diff
@@ -7,5 +7,5 @@ property.dbTypes.displayName=Type de base de donn\u00E9es
 property.jdbcUrl.displayName=URL JDBC
 property.driverClass.displayName=Pilote
 
-property.userId.displayName=Nom d''utilisateur
+property.userId.displayName=Nom d'utilisateur
```
Comment (gonchar-ivan) [inline components/components-jdbc/components-jdbc-definition/src/main/resources/org/talend/components/jdbc/datastore/messages_fr.properties]: add quote
```suggestion
property.userId.displayName=Nom d''utilisateur
```

File: components/components-marklogic/components-marklogic-definition/src/main/resources/org/talend/components/marklogic/tmarklogicconnection/MarkLogicConnectionProperties_fr.properties [hunk 7]
```diff
@@ -1,7 +1,7 @@
 property.host.displayName=H\u00F4te
 property.port.displayName=Port
 property.database.displayName=Base de donn\u00E9es
-property.username.displayName=Nom d''utilisateur
+property.username.displayName=Nom d'utilisateur
```
Comment (gonchar-ivan) [inline components/components-marklogic/components-marklogic-definition/src/main/resources/org/talend/components/marklogic/tmarklogicconnection/MarkLogicConnectionProperties_fr.properties]: add quote
```suggestion
property.username.displayName=Nom d''utilisateur
```

File: components/components-salesforce/components-salesforce-definition/src/main/resources/org/talend/components/salesforce/datastore/messages_fr.properties [hunk 8]
```diff
@@ -1,9 +1,9 @@
 definition.SalesforceDatastore.title=Modifier la connexion \u00E0 Salesforce
 definition.SalesforceDatastore.displayName=Salesforce
-form.Main.title=Connexion Salesforce
-form.Main.displayName=Connexion Salesforce
+form.Main.title=Modifier la connexion \u00E0 Salesforce
+form.Main.displayName=Modifier la connexion \u00E0 Salesforce
 
-property.userId.displayName=Nom d''utilisateur
+property.userId.displayName=Nom d'utilisateur
```
Comment (gonchar-ivan) [inline components/components-salesforce/components-salesforce-definition/src/main/resources/org/talend/components/salesforce/datastore/messages_fr.properties]: add quote
```suggestion
property.userId.displayName=Nom d''utilisateur
```

File: core/components-common-oauth/src/main/resources/org/talend/components/common/oauth/messages_fr.properties [hunk 9]
```diff
@@ -4,5 +4,5 @@ form.Main.displayName=Principal
 msg.info.showAuthorizUrl=Collez cette URL dans un navigateur Web pour autoriser l'acc\u00E8s\u00A0:
```
Comment (gonchar-ivan) [inline core/components-common-oauth/src/main/resources/org/talend/components/common/oauth/messages_fr.properties]: add quote
```suggestion
msg.info.showAuthorizUrl=Collez cette URL dans un navigateur Web pour autoriser l''acc\u00E8s\u00A0:
```

File: core/components-common/src/main/resources/org/talend/components/common/BulkFileProperties_fr.properties [hunk 10]
```diff
@@ -1,5 +1,5 @@
-property.bulkFilePath.displayName=Chemin du fichier
-property.append.displayName=\u00C9crire apr\u00E8s
+property.bulkFilePath.displayName=Chemin d'acc\u00E8s au fichier
```
Comment (gonchar-ivan) [inline core/components-common/src/main/resources/org/talend/components/common/BulkFileProperties_fr.properties]: add quote
```suggestion
property.bulkFilePath.displayName=Chemin d''acc\u00E8s au fichier
```

**Comment by build-talend-common** [comment]
Pull request test result: [failure](https://ci-common.datapwn.com/job/components_pull-request_master/2/) [:x:](https://ci-common.datapwn.com/job/components_pull-request_master/2/)

**Comment by build-talend-common** [comment]
Pull request test result: [failure](https://ci-common.datapwn.com/job/components_pull-request_master/4/) [:x:](https://ci-common.datapwn.com/job/components_pull-request_master/4/)

**Pull request merged**